app/src/main/AndroidManifest.xml

```diff
@@ -8,12 +8,18 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.ParkingSystem">
-        <activity android:name=".MainActivity">
+        <activity android:name=".activities.MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".activities.ParkingSpaceReservationActivity"
+            android:parentActivityName=".activities.MainActivity">
```

Comment:
why do you add this property? it is not necessary

Reply:
I added because I read that, with this property, adds to the activity (in this case ParkingSpaceReservationActivity) a back button to the parent activity (MainActivity). I thought it was a good idea

If I can keep this, I will remove the meta tag because it is needed only if my API support level 15 or lower